repository-data/webfiles/src/main/resources/site/freemarker/common/search-banner.ftl

```diff
@@ -0,0 +1,28 @@
+<#ftl output_format="HTML">
+<#include "../include/imports.ftl">
+
+<@hst.setBundle basename="homepage.website.labels"/>
+<@fmt.message key="search-banner.placeholder" var="placeholderText"/>
+<@fmt.message key="search-banner.buttonLabel" var="buttonLabel"/>
+<@fmt.message key="search-banner.title" var="searchTitle"/>
+<@fmt.message key="search-banner.text" var="searchBannerText"/>
+
+<#if facets??>
```

Comment:
this is already in `../include/imports.ftl`

Reply:
@wojtek-oledzki Thanks for the heads up, I'll remove it.